python/lsst/sims/catUtils/mixins/VariabilityMixin.py

```diff
@@ -226,6 +237,9 @@ def applyMicrolens(self, params, expmjd_in):
 
     @register_method('applyAgn')
     def applyAgn(self, params, expmjd_in):
```

Comment:
Sounds good .... While reviewing this I was also noticing sncat is missing some docstrings, maybe this should be added to the ticket as well.